docs/apache-hive-catalog.md

```diff
@@ -40,6 +40,8 @@ The Hive catalog supports creating, updating, and deleting databases and tables
 | `kerberos.check-interval-sec`            | The interval to check validness of the principal                                                                                                                                                                  | 60            | No                           | 0.4.0         |
 | `kerberos.keytab-fetch-timeout-sec`      | The timeout to fetch key tab                                                                                                                                                                                      | 60            | No                           | 0.4.0         |
 
+When you use the Gravitino in Trino. You can pass the Trino Hive connector configuration using prefix `trino.bypass`. For example using `trino.bypass.hive.config.resources` to passing the `hive.config.resources` to the Gravitino Hive catalog in Trino runtime.
```

Comment:
I think it would be better "When you use Gravitino with Trino...". Besides, it is a bit strange to put it here, not in Trino's doc?

Also, "For example, using xxx to pass..."

Reply:
If it's not explained here, users might see `trino.bypass.hive.config.resources` in the catalog properties and decide to remove it.